infrastructure/environments/cloudformation/full/organisation/dagster.yaml

```diff
@@ -80,20 +80,37 @@ Parameters:
     Type: String
     Default: "repository.py"
     Description: Location of the repo.py file that controls the repository
+
+  # Dagster Parameters
   InputLocation:
     Type: String
     Description: |
       The standard location to look for files to ingest into the pipelines. For s3, this must be a ROOT path, no folders
       Example: s3://my-bucket-name
+  InputLocationArn:
+    Type: String
+    Description: ARN of the InputLocation bucket
   OutputLocation:
     Type: String
     Description: The standard output location for pipeline files. For s3, this must be a ROOT path, no folders.
+  WorkspaceLocation:
+    Type: String
+    Description: The standard output location for pipeline files. For s3, this must be a ROOT path, no folders.
   ExternalDataLocation:
     Type: String
     Description: The location to look for data files coming from external sources (e.g. ONS, post codes, etc)
-  OfstedExternalDataLocation:
+  CleanPipelineSchedule:
     Type: String
-    Description: The location to look for the Ofsted data (external resource)
+    Description: The CRON string used to determine how often the clean file pipeline should run
```

Comment:
Should this have the comment from below about never running on the ORG instance?

Reply:
Might be a good idea. Will add.